client/cli/src/lib.rs

```diff
@@ -47,13 +45,11 @@ use structopt::{
 	clap::{self, AppSettings},
 	StructOpt,
 };
-use tracing_subscriber::{
-	filter::Directive, fmt::time::ChronoLocal, layer::SubscriberExt, FmtSubscriber, Layer,
-};
-
-pub use logging::PREFIX_LOG_SPAN;
+pub use sc_logging::{PREFIX_LOG_SPAN, prefix_logs_with};
```

Comment:
I moved the macro to sc-logging too because it makes more sense.

Here I'm re-exporting the macro in sc-cli. We can revert this easily if you don't like it.

Reply:
Probably don't need it, so this can be removed.

Reply:
Done... somewhere